spec/API_specification/array_api/array_object.py

```diff
@@ -493,17 +520,34 @@ def __index__(self: array, /) -> int:
 
     def __int__(self: array, /) -> int:
         """
-        Converts a zero-dimensional integer array to a Python ``int`` object.
+        Converts a zero-dimensional array to a Python ``int`` object.
+
+        .. note::
+           When casting a boolean input array, a value of ``True`` must cast to ``1``, and a value of ``False`` must cast to ``0``.
+
+        **Special cases**
+
+        For floating-point operands,
+
+        - If ``self`` is a finite number, the result is the integer part of ``self``.
```

Comment:
We should probably be pedantic here and say what happens for `-0`.

Reply:
Ah good shout. Added a special case—thought this didn't warrant a note (either inline or a Sphinx directive), but not too sure on that.